lms/djangoapps/certificates/models.py

```diff
@@ -206,6 +205,7 @@ class GeneratedCertificate(models.Model):
     """
     Base model for generated certificates
     """
+    from course_modes.models import CourseMode
```

Comment:
Could you include a comment as to why this import needs to be done here instead of at the top of the file